pkg/nsx/services/securitypolicy/builder.go

```diff
@@ -1932,3 +1912,36 @@ func (service *SecurityPolicyService) BuildNetworkPolicyAllowPolicyID(uid string
 func (service *SecurityPolicyService) BuildNetworkPolicyIsolationPolicyID(uid string) string {
 	return strings.Join([]string{uid, common.RuleActionDrop}, common.ConnectorUnderline)
 }
+
+type namedPortInfo struct {
+	port v1alpha1.SecurityPolicyPort
+	ips  []string
+
+	// idSuffix is used in T1 environment to generate the NSX rule ID. It is constructed by
+	// SecurityPolicyPortIdx_PortAddressIdx.
+	// TODO: Remove this field after we don't use the SecurityPolicyPort index in NSX rule ID with T1 topology.
+	idSuffix string
+}
+
+func newNamedPortInfo(port v1alpha1.SecurityPolicyPort) *namedPortInfo {
```

Comment:
This func is actually for non named port case, 
how about rename to `newPortInfo`